packages/ui/src/components/NavBar/index.tsx

```diff
@@ -25,7 +23,7 @@ const NavBar = ({ title }: Props) => {
         }}
       >
         <ArrowPrev />
-        {!isMobile && <span>{t('action.nav_back')}</span>}
+        <span>{t('action.nav_back')}</span>
```

Comment:
just double check removing `!isMobile` check is by design

Reply:
yes, using style display none on mobile